### PR TITLE
Clean up pinata

### PR DESCRIPTION
**Pull request by tomquirk**
Closes https://github.com/jbx-protocol/juice-interface/issues/2585

Pinata is no longer in use across the site. This PR cleans up any unused code.

Infura has no rate limit issues (150 reqs per second) so I don't think we need the complexities of sign-to-upload.

Genuinely sad and sorry to be removing your hard work here @wraeth-eth. It's very good work and we can def draw from this gained knowledge in the future. I packaged it up in this PR so that we have a reference point if we ever need to come back to it.

LMKWYT.

### CODE REVIEW

File: src/utils/ipfs.ts [hunk 1]
```diff
@@ -3,24 +3,14 @@ import { base58 } from 'ethers/lib/utils'
...
 /**
  * Return a URL to our open IPFS gateway for the given cid USING INFURA.
  *
  * The 'open' gateway returns any content that is available on IPFS,
  * not just the content we have pinned.
  */
-export const ipfsOpenGatewayUrl = (cid: string | undefined): string => {
-  return ipfsGatewayUrl(cid, OPEN_IPFS_GATEWAY_HOSTNAME)
+export const ipfsGatewayUrl = (cid: string | undefined): string => {
+  return `https://${OPEN_IPFS_GATEWAY_HOSTNAME}/ipfs/${cid}`
```
Comment (wraeth-eth) [inline src/utils/ipfs.ts:13]: Will we need to change things locally?

File: src/utils/server/ipfs/infuraScheduler.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import Bottleneck from 'bottleneck'
+
+// Back off for 3 secs - pinata rate limits for 180/min
```
Comment (wraeth-eth) [inline src/utils/server/ipfs/infuraScheduler.ts:3]: Change these settings to match infura IPFS backoff

**Review by wraeth-eth — APPROVED** [review]
@tomquirk nah this is great! you misunderestimate how much I love deleting code

File: src/utils/ipfs.ts (hunk 1 above)
Comment (tomquirk) [reply src/utils/ipfs.ts:13]: nup all g

File: src/utils/server/ipfs/infuraScheduler.ts (hunk 2 above)
Comment (tomquirk) [reply src/utils/server/ipfs/infuraScheduler.ts:3]: ah good catch, done!

**Pull request merged**